mlos_bench/mlos_bench/services/remote/azure/README.md

```diff
@@ -0,0 +1,37 @@
+# Azure Services
+
+## Authentication
+
+This service allows us to generate tokens for authenticating with other Azure services.
+It supports authenticating as the following identities:
+
+### User
+
+As a user, the only requirement is that a user is logged in via the Azure CLI before `mlos_bench` is started i.e. `az login` or `az login --device-login`.
+This method is simpler to get started, but the drawback is that organization policies might limit the period in which we can generate access tokens under a user logged in via the CLI.
+This would lead to early termination of an experiment once tokens cannot be generated anymore without another interactive login by the user.
```

Comment:
```suggestion
This would lead to early termination of an experiment once tokens cannot be generated anymore without another interactive login by the user.

To avoid this issue, we also support automated authentication as a Service Principle, described next.
```